File: labs/07 Access Control/web-login/php-apache/html/v5/break/break.php
<?php

$DICTIONARY = "words.txt";

// enter the DB hash below
$db_hash = "";

// grab the dictionary words
$words = file($DICTIONARY, FILE_IGNORE_NEW_LINES);

// hash each dictionary word and compare
foreach ($words as $password)
{
    echo "Trying: $password\n";
    if (password_verify($password, $db_hash))
    {
        // we found it!
        echo "**SUCCESS!\n";
        break;
    }
}

?>

File: labs/07 Access Control/web-login/php-apache/html/v5/break/break.py
from bcrypt import checkpw

DICTIONARY = "words.txt"

# enter the DB hash below
db_hash = "";

# grab the dictionary words
with open(DICTIONARY, "r") as f:
    words = f.read().splitlines()

# hash each dictionary word and compare
for password in words:
    print(f"Trying: {password}")
    if (checkpw(password.encode("utf-8"), db_hash.encode("utf-8"))):
        # we found it!
        print("**SUCCESS!")
        break
